client/layout/masterbar/style.scss

```diff
@@ -891,6 +891,14 @@ a.masterbar__quick-language-switcher {
 		@include breakpoint-deprecated( "<480px" ) {
 			display: none;
 		}
+
+		&.is-active {
+			background: transparent;
```

Comment:
I'd suggest applying `--color-checkout-masterbar-item-hover-background` for this state.

Notice how the active state for the masterbar elsewhere has a similar pattern when the Help Center is open.

<img width="447" alt="Screenshot 2023-07-16 at 3 19 12 PM" src="https://github.com/Automattic/wp-calypso/assets/1126811/be125b9b-f13b-4b8f-975a-54bcff0c4a17">

Reply:
Nice thanks!